analyzer/src/main/kotlin/managers/Pub.kt

```diff
@@ -75,9 +76,9 @@ private const val GRADLE_VERSION = "5.6.4"
 private const val PUB_LOCK_FILE = "pubspec.lock"
 
 private val flutterCommand = if (Os.isWindows) "flutter.bat" else "flutter"
-private val pubCommand = if (Os.isWindows) "pub.bat" else "pub"
+private val dartCommand = if (Os.isWindows) "dart.bat" else "dart"
```

Comment:
This actually needs to be `dart.exe` on Windows (which causes the test failure),

Reply:
The downloaded package also contains the `dart.bat` file.
I'll try your suggestion, but I don't really understand why this is required.

Reply:
At least the Windows package I downloaded only contains a `dart.exe` in the path where `dart.bat` is being looked for.

Reply:
Here's the content that I see:

```
 ~/Downloads/flutter_windows_2.2.3-stable/flutter (stable)
$ find . -type f -name "dart.exe" -or -name "dart.bat"
./bin/cache/dart-sdk/bin/dart.exe
./bin/dart.bat
```

Reply:
And because of

    private val pubAbsolutePath = flutterAbsolutePath.resolve("cache/dart-sdk/bin")

the path being used is "bin/cache/dart-sdk/bin", not just "bin".